src/object/SConscript

```diff
@@ -10,7 +10,8 @@ def scons():
     denv = env.Clone()
 
     # Common object code
-    common_tgts = denv.SharedObject(['obj_class.c', 'obj_rpc.c', 'obj_task.c'])
+    common_tgts = denv.SharedObject(['obj_class.c', 'obj_rpc.c', 'obj_task.c',
+				     'obj_utils.c'])
```

Comment:
(pylint-bad-continuation) Wrong continued indentation (add 28 spaces).

Reply:
Just add one space before 'obj_utils.c'.  It is supposed to line up with entries in the python array